Edited instruction to make it easier to understand (#34577)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/use-the-s-tag-to-strikethrough-text.english.md

## Instructions
<section id='instructions'>

> Wrap the <code>s</code> tag around "Google" that is inside the <code>h4</code> tag, and then add beside it the word "Alphabet", which should not have the strikethrough formatting.

Reviewer: @ceejayoz I think the only change you provided which could improve the readability might be the addition of `that is`.  The addition the comma is not valid comma use and reordering `beside it` make it much for confusing.
What do you think about the following suggestion?  It could be changed if you do not find it helps.
```suggestion
Wrap the <code>s</code> tag around "Google" that is inside the <code>h4</code> element and add a space and the word "Alphabet" after the closing <code>s</code> tag , which should not have the strikethrough formatting.
```

Reviewer 2: @RandellDawson Hey, think you accidentally tagged the wrong person.

Reviewer: @ceejayoz Sorry about that.  Usually when you type the `@` to tag someone, it brings up the OP, but for some reason your username came up instead and I did not notice it.
